src/main/java/it/gov/pagopa/pu/classification/model/Assessments.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.classification.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.io.Serializable;
+
+@Entity
+@Table(name = "assessments")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Assessments extends BaseEntity implements Serializable {
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "assessments_generator")
+  @SequenceGenerator(name = "assessments_generator", sequenceName = "assessment_id_seq", allocationSize = 1)
+  private Long assessmentId;
+  @NotNull
+  private Long organizationId;
+  @NotNull
+  private String debtPositionTypeOrgCode;
+  @NotNull
+  private String status;
+  @NotNull
+  private String assessmentName;
+  @NotNull
+  private Boolean printed = false;
```

Comment:
if not null and default false, we could use directly the native type
```suggestion
  private boolean printed;
```